### DISCUSSION TITLE
v0.4.1

### DISCUSSION THREAD

**Discussion by tdrkDev** [HandyGram/app#4] [Announcements]
Changelog (from 0.4.0):
• Sending text, voice, sticker, GIF messages
• Removing and adding reactions, reactions chooser
• Seeing chat, user, channel info
• Deleting messages for everyone and only for yourself
• Synchronous updates
• Messages are now marked as viewed if they're shown on screen
• On-screen clock
• UI settings - font scale, hiding profile pictures, showing clock and more
• Fixed exiting from home page with back button
• Fixed minor bugs
• Many other changes which I've forgot to write there
• Doubled the app size 🤡

0.4.1 is a bug-fix release, so there are no big changes like in 0.4.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/HandyGram/HandyGram/releases/tag/v0.4.1'>v0.4.1</a>.</em>

**Comment by LK024** [comment]
Hi, Handygram works really well to be in its very first versions, for now I have only found 2 flaws:
1) I cannot use the Gesture Back of the watch by scrolling from the left edge but only with the button (it would have been interesting to be able to deactivate the button and instead use the gesture)
2) In watches like my Google Pixel Watch equipped with a crown to scroll through the menus, turning it the chat flows too much making it practically unusable

**Reply by LK024** [reply]
Update, I've formatted the Watch and now I can't login becouse the connirm password button doesn't work
